tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNetCore/UserEvents/IIdentityUser.cs

```diff
@@ -0,0 +1,21 @@
+﻿// <copyright file="IIdentityUser.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+#nullable enable
+using System.ComponentModel;
+
+namespace Datadog.Trace.ClrProfiler.AutoInstrumentation.AspNetCore.UserEvents;
+
+/// <summary>
+/// aspnet core IApplicationBuilder
```

Comment:
NIT. Same here

Reply:
fixed in 15c46cfa3b82b526f1bed4c2d3ec2808ad984f45